feedback.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+
+<head>
+    <title>Feedback</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <link rel="stylesheet" type="text/css" href="MemoryGame.css">
+</head>
+
+<body>
+    <div id="navbar">
+        <span class="heading"><img style="margin-left: -8px" class="logo" src="./assets/signtogether_centered.png"></span>
+    </div>
+    <div class="main">
+        Gerne kannst du eine Mail an agnes.koehler@codecentric.de mit deinen Verbesserungsvorschlägen und Ideen schreiben!
```

Comment:
We also have the "+" Button that currently opens an `alert`. Perhaps replace that with a link to this page?

Reply:
The + button could possibly have the possibility to add a new category in the future. So currently this button is a test if people even find it or want to use that feature. So maybe now it is more or less the same as the feedback page, but it serves a different purpose.